x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1172,6 +1264,36 @@ var security = (function () {
         evt.Put("winlog.event_data.UserAccountControl",uac_list);
       };
 
+    var addAuditInfo = function(evt) {
+        var subcategoryGuid = evt.Get("winlog.event_data.SubcategoryGuid").replace("{",'').replace("}",'').toUpperCase();
+        if (!subcategoryGuid) {
+            return;
+        }
+        if (!audit_description[subcategoryGuid]) {
+            return;
+        }
+        evt.Put("winlog.event_data.Category",audit_description[subcategoryGuid][1]);
+        evt.Put("winlog.event_data.SubCategory",audit_description[subcategoryGuid][0]);
+        var coded_actions=evt.Get("winlog.event_data.AuditPolicyChanges").split(",");
+        var action_results=[];
+        for (var j=0; j<coded_actions.length; j++) {
+            var action_code=coded_actions[j].replace("%%",'').replace(' ','');
+            action_results.push(audit_actions[action_code]);
+       }
+       evt.Put("winlog.event_data.AuditPolicyChangesDescription",action_results);
+    };
+
+    var addRelatedUser= function(evt,user) {
+        var related_user = evt.Get("related.user");
+        if (!related_user) {
```

Comment:
From and Elasticsearch document standpoint this is not necessary. You can just call `evt.AppendTo` in both cases and it will work. Whether the value is a string or an array or strings it's going to be handled the same way in ES.

Reply:
I have modified the code in order to use directly the evt.AppentTo(related.user,user_to_add), less functions calls, less code, more efficiency :)
Thank you for the suggestion. I'll have this in mind for futures PRs

Reply:
@andrewkroh some checks fail regarding to the documentation update.
I have added the events ID in \beats\x-pack\winlogbeat\module\security\_meta\docs.asciidoc.
Shall I do something more?

![image](https://user-images.githubusercontent.com/33020901/73177284-e543e100-410e-11ea-995d-f3ce4d548216.png)